jest.config.js

```diff
@@ -44,6 +44,6 @@ module.exports = {
   verbose: false,
   testPathIgnorePatterns: ['<rootDir>/packages/visx-demo'],
   transformIgnorePatterns: [
-    'node_modules/(?!(d3-(array|color|format|interpolate|scale|time|time-format)|internmap)/)',
+    'node_modules/(?!(d3-(array|color|delaunay|format|interpolate|scale|time|time-format)|delaunator|internmap|robust-predicates)/)',
```

Comment:
ah interesting, I missed this in the esm/cjs work. I might need to revisit this (in a separate PR)

Reply:
I had another look at this, and I was able to remove `d3-delaunay` from here. The other two still seem to be necessary.

Reply:
Thanks for looking at this. I did some more digging and it seems like `delaunator` and `robust-predicates` may be ESM-only which likely trips up jest. 

I pulled and built your branch locally and you can see that this generated/vendored CJS file references an the ESM-only `delaunator` lib. I'm not sure why that doesn't mess up our next demo app, perhaps it's all resolving to ESM. 

all this to say that if someone hits CJS/ESM issues with this we might have to also vendor `delaunator` + `robust-predicates` in the future, but since it seems to work with `next.js` as-is I think we can merge this for now.

Reply:
<img width="692" alt="image" src="https://github.com/airbnb/visx/assets/4496521/620b38cc-fc7e-4141-af8e-c6c58fc7ff29">